app/src/main/java/edu/sharif/prj01/rock_paper_scissor/Player.java

```diff
@@ -0,0 +1,51 @@
+import java.util.Random;
+
+public class Player extends Thread {
+    private Weapon weapon;
+
+    public Player() {
+        move();
+    }
+
+    public Weapon getWeapon() {
+        return weapon;
+    }
+
+    @Override
+    public void run() {
+        while (true) {
+            try {
+                Thread.sleep(new Random().nextInt(1000) + 1000);
```

Comment:
what is the application of this line?

Reply:
It sets a random delay to prevent the players to run very fast. And makes the output readable.